docs/man_pages/project/testing/livesync.md

```diff
@@ -0,0 +1,32 @@
+livesync
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ tns livesync [<Command>]`
+
+Synchronizes the latest changes in your project to devices. 
+`<Command>` extends the `livesync` command. You can set the following values for this attribute.
```

Comment:
`<Platform>` is the target mobile platform to which you want to synchronize your changes. You can set the following target platforms.